specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 23 Oct 2020 21:33:05 GMT",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-subscription-reads": "14909",
+        "x-ms-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b",
+        "x-ms-correlation-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.Synapse/workspaces/exampleWorkspaceName/libraryartifacts/exampleLibraryName1.jar",
+            "name": "exampleLibraryName1.jar",
+            "type": "Microsoft.Synapse/workspaces/libraryartifacts",
+            "properties": {
+              "name": "exampleLibraryName1.jar",
```

Comment:
name should not be repeated inside the property bag.  "type" should also not be named "type" within the property bag as that is a reserved word for the top level type.

Reply:
This is how we are getting the response from the partner ADF team. Also, we have another data-plane API that follows the same schema. In addition, our partner teams(portal and studio) requested the response in this format.

Reply:
This is not a data-plane API.  Control plane ARM APIs need to follow ARM API conventions to maintain consistency across Azure.